src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessarySemicolonInTryWithResourcesCheck.java

```diff
@@ -0,0 +1,133 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2019 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+import com.puppycrawl.tools.checkstyle.StatelessCheck;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+/**
+ * <p>
+ * Checks if unnecessary semicolon is used in last resource declaration.
+ * </p>
+ * <ul>
+ * <li>
+ * Property {@code allowWhenNoBraceAfterSemicolon} -
+ * Allow unnecessary semicolon if closing paren is not on the same line.
+ * Default value is {@code false}.
```

Comment:
we need to update issue, looks like we lost this during final review of design.

default value should be "true".
as I want cases like:
```
          try(Reader r6 = new PipedReader();
              Reader r7 = new PipedReader(); // NO violation
          ){}
```
As it is friendly for minimization of diff.

Reply:
done.

Original issue is updated, docs and javadoc as well.